README.md

```diff
@@ -14,6 +14,10 @@
 
 loaders.gl is extensively documented on the [loaders.gl](https://loaders.gl) website.
 
+## Contributing 
+
+See [CONTRIBUTING.md].
+
```

Comment:
Adding link to contributing guide here for visibility. I initially didn't see it in the README and thought I was supposed to reference the visgl/deckgl contributing guide

Reply:
Does a markdown link like that (without parenthesis following the square brackets) actually work? When I view this file in github, it does not. 

Either drop the square parenthesis, or add a link?

Reply:
Ah yes, you're right. my bad, will fix